### PR TITLE
Interface

### PR DESCRIPTION
**Pull request by yliu0715**
Review da code :)

### CODE REVIEW

File: chess-bot/interface.py [hunk 1]
```diff
@@ -1,7 +1,54 @@
...
+
+    def get_data(self):
+        """Returns data recevied through websocket
+        """
+        signal.signal(signal.SIGALRM,self.timeout_handler)
+        while True:
+            signal.alarm(2)
+            try:
+                data = self.ws_connection.recv()
+                content = self.get_fenstring(data)
```
Comment (ChaosDL) [inline chess-bot/interface.py]: put line 46, 47, 48 into else

Comment (ChaosDL) [reply chess-bot/interface.py]: +                content = self.get_fenstring(data) 
+                if content:
+                    return content

Comment (yliu0715) [reply chess-bot/interface.py]: Yes I see it

Comment (ChaosDL) [reply chess-bot/interface.py]: After the alarm reset

**Comment by ChaosDL** [comment]
Good job  👍

**Pull request merged**